Failures seem relevant:
```
ERROR: corehq.apps.app_manager.tests.test_views:TestViews.test_basic_app
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/vendor/lib/python2.7/site-packages/mock.py", line 1224, in patched
    return func(*args, **keywargs)
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/tests/test_views.py", line 163, in test_basic_app
    self._test_status_codes(['view_form', 'form_source'], kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/tests/test_views.py", line 120, in _test_status_codes
    response = self.client.get(reverse(name, kwargs=kwargs), follow=follow)
  File "/vendor/lib/python2.7/site-packages/django/test/client.py", line 529, in get
    **extra)
  File "/vendor/lib/python2.7/site-packages/django/test/client.py", line 333, in get
    return self.generic('GET', path, secure=secure, **r)
  File "/vendor/lib/python2.7/site-packages/django/test/client.py", line 409, in generic
    return self.request(**r)
  File "/vendor/lib/python2.7/site-packages/django/test/client.py", line 494, in request
    six.reraise(*exc_info)
  File "/vendor/lib/python2.7/site-packages/django/core/handlers/exception.py", line 42, in inner
    response = get_response(request)
  File "/vendor/lib/python2.7/site-packages/django/core/handlers/base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "/vendor/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/vendor/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/vendor/lib/python2.7/site-packages/django/views/decorators/http.py", line 40, in inner
    return func(request, *args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/domain/decorators.py", line 103, in _inner
    return view_func(req, domain_name, *args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/views/forms.py", line 710, in view_form
    return view_generic(request, domain, app_id, module_id, form_id)
  File "submodules/dimagi-utils-src/dimagi/utils/couch/resource_conflict.py", line 24, in new_fn
    return fn(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/views/view_generic.py", line 99, in view_generic
    if (form is not None and toggles.USER_PROPERTY_EASY_REFS.enabled(domain)
  File "/mnt/commcare-hq-ro/corehq/toggles.py", line 80, in enabled
    and was_domain_created_after(item, enabled_after)):
  File "/mnt/commcare-hq-ro/corehq/toggles.py", line 144, in was_domain_created_after
    return domain_obj is not None and domain_obj.date_created > checkpoint
TypeError: can't compare datetime.datetime to NoneType
```

and 

```
ERROR: corehq.apps.app_manager.tests.test_util:DisabledUserPropertiesSchemaTest.test_get_casedb_schema
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/vendor/lib/python2.7/site-packages/mock.py", line 1224, in patched
    return func(*args, **keywargs)
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/tests/test_util.py", line 425, in test_get_casedb_schema
    schema = util.get_casedb_schema(form)
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/util.py", line 454, in get_casedb_schema
    if is_usercase_in_use(app.domain) and toggles.USER_PROPERTY_EASY_REFS.enabled(app.domain):
  File "/mnt/commcare-hq-ro/corehq/toggles.py", line 80, in enabled
    and was_domain_created_after(item, enabled_after)):
  File "/mnt/commcare-hq-ro/corehq/toggles.py", line 143, in was_domain_created_after
    domain_obj = Domain.get_by_name(domain)
  File "/vendor/lib/python2.7/site-packages/quickcache/quickcache.py", line 40, in inner
    return helper(*args, **kwargs)
  File "/vendor/lib/python2.7/site-packages/quickcache/quickcache_helper.py", line 145, in __call__
    return self.call(*args, **kwargs)
  File "/vendor/lib/python2.7/site-packages/quickcache/quickcache_helper.py", line 60, in call
    content = self.fn(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/domain/models.py", line 574, in get_by_name
    domain = _get_by_name(stale=(not strict))
  File "/mnt/commcare-hq-ro/corehq/apps/domain/models.py", line 566, in _get_by_name
    result = cls.view("domain/domains", key=name, reduce=False, include_docs=True, **extra_args).first()
  File "/vendor/lib/python2.7/site-packages/couchdbkit/schema/base.py", line 310, in view
    return db.view(view_name,
  File "/vendor/lib/python2.7/site-packages/mock.py", line 700, in __getattr__
    raise AttributeError("Mock object has no attribute %r" % name)
AttributeError: Mock object has no attribute 'view'
```

--- a/corehq/apps/app_manager/tests/test_util.py
+++ b/corehq/apps/app_manager/tests/test_util.py
@@ -1,3 +1,4 @@
+from datetime import datetime
 import corehq.apps.app_manager.util as util
 from corehq.apps.app_manager.models import (
     Module,
@@ -405,6 +406,8 @@ class SchemaTest(SimpleTestCase):
 @patch('corehq.apps.app_manager.models.is_usercase_in_use', MagicMock(return_value=True))
 @patch('corehq.apps.app_manager.util.is_usercase_in_use', MagicMock(return_value=True))
 @patch('corehq.apps.app_manager.util.get_per_type_defaults', MagicMock(return_value={}))
+@patch('corehq.apps.domain.models.Domain.get_by_name',
+    staticmethod(lambda name: MagicMock(spec=["date_created"], date_created=datetime(2017, 1, 1))))
 class DisabledUserPropertiesSchemaTest(SimpleTestCase):
     # TODO remove this test when removing USER_PROPERTY_EASY_REFS toggle
 

--- a/corehq/toggles.py
+++ b/corehq/toggles.py
@@ -141,7 +141,11 @@ def was_domain_created_after(domain, checkpoint):
     """
     from corehq.apps.domain.models import Domain
     domain_obj = Domain.get_by_name(domain)
-    return domain_obj is not None and domain_obj.date_created > checkpoint
+    return (
+        domain_obj is not None and
+        domain_obj.date_created is not None and
+        domain_obj.date_created > checkpoint
+    )
 
 
 def deterministic_random(input_string):

